reana_client/cli/files.py

```diff
@@ -161,6 +162,13 @@ def download_files(ctx, workflow, file_, output_directory, access_token):
                         fg='red'), err=True)
         sys.exit(1)
 
+    if not file_:
+        reana_spec = load_reana_spec(os.path.join(get_workflow_root(),
+                                     'reana.yaml'),
+                                     False)
+        if 'outputs' in reana_spec:
+            file_ = reana_spec['outputs'].get('files') or []
+
     if workflow:
```

Comment:
Please enrich CLI documentation and ideally also the "User guide" with a few words, see https://github.com/reanahub/reana-client/pull/181#issuecomment-422451242

(Could also add a note that only output file download is supported for now, not directories, there)